posts/2021-05-15-Rustup-1.24.2.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Announcing Rustup 1.24.2"
+author: The Rustup Working Group
+---
+
+The rustup working group is happy to announce the release of rustup version 1.24.2. [Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.24.2 is as easy as closing your IDE and running:
+
+```
+rustup self update
+```
+
+Rustup will also automatically update itself at the end of a normal toolchain update:
+
+```
+rustup update
+```
+
+If you don't have it already, you can [get rustup][install] from the appropriate page on our website.
+
+[rust]: https://www.rust-lang.org
+[install]: https://rustup.rs
+
+## What's new in rustup 1.24.2
+
+The primary change in 1.24.2 is further refinement of the work we have been doing
```

Comment:
This feels waffly to me :).

1.24.2 introduces pooled allocations to prevent memory fragmentation issues on some platforms with 1.24.0/1.24.1. We're not entirely sure what aspect of the streamed unpacking logic caused allocator fragmentation, but memory pools are a well known fix that should solve this for all platforms.